### DISCUSSION TITLE
Welcome to discloud-api Discussions!

### DISCUSSION THREAD

**Discussion by pedroricardo** [discloud/discloud-api#36] [Announcements]
## 👋 Bem-vindo!
  Estamos usando as Discussões como um lugar para nos conectarmos com outros membros da nossa comunidade. Esperamos que você:
  * Faça perguntas sobre as quais você está se perguntando.
  * Compartilhar ideias.
  * Envolva-se com outros membros da comunidade.
  * Dê boas-vindas aos outros e tenha a mente aberta. Lembre-se que esta é uma comunidade que
  construir juntos 💪.

  Para começar, comente abaixo com uma apresentação sua e conte-nos sobre o que você faz com esta comunidade.

**Comment by h3xjs** [comment]
```c#

```
